terraform/vultr/README.md

```diff
@@ -0,0 +1,95 @@
+# Deploying strfry to vultr using terraform
+
+## Install terraform
+
+ 1. Add the Terraform GPG key to your server
+
+    ' $ curl -fsSL https://apt.releases.hashicorp.com/gpg | sudo apt-key add -'
+
+ 2. Add the official Terraform repository to your APT sources
+
+    ' $ sudo apt-add-repository "deb [arch=amd64] https://apt.releases.hashicorp.com focal main"'
+
+ 3. Update the server packages
+
+    ' $ sudo apt update'
+
+ 4. Install Terraform on the server
+
+    ' $ sudo apt install terraform'
+
+
+## Activate Your Vultr API Key
+
+Activate and Copy your Vultr API Key from the Vultr [Customer Portal Settings Page](https://my.vultr.com/settings/#settingsapi)
+
+## Edit Configuration Files
+
+1. Edit terraform.tfvars (see terraform.tfvars.example)
+   
+    ```
+    VULTR_API_KEY   = "EGJGEJIGJKSDGJKSDKSDGJKLDG444JLKG"
```

Comment:
This is not a live key right?

Reply:
no, but i change this to a less alarming example